<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eulaForm</class>
 <widget class="QWidget" name="eulaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AntiZTools用户协议</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>500</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>323</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agreeBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>323</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>同意</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
